src/lua/api-gateway/validation/oauth2/oauthTokenValidator.lua

```diff
@@ -166,15 +166,8 @@ function _M:getTokenFromCache(cacheLookupKey)
     return nil;
 end
 
--- imsAuth will validate the service token passed in "Authorization" header --
-function _M:validate_ims_token()
+function _M:validateOAuthToken(oauth_token, validation_config)
```

Comment:
I didn't think so much of the `validateOAuthToken` being public. But it makes perfect sense to use it from a composite validator, so I can then revert the `exitFn` call back to the validator class.
I will update the code.